components/automate-ui/src/app/components/create-object-modal/create-object-modal.component.ts

```diff
@@ -21,10 +21,9 @@ export class CreateObjectModalComponent implements OnInit, OnDestroy, OnChanges
   @Input() visible = false;
   @Input() creating = false;
   @Input() objectNoun: string;
-  @Input() createProjectModal = false;
```

Comment:
Refinement: co-dependent parameter converted to local variable to encapsulate it.

Reply:
Does it need to be public? I fail to see the reason just now.